cartopy/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - numpy x.x
     - cython
     - proj.4
-    - geos
+    - geos 3.4.*
```

Comment:
This is a good move because `defaults` updated to geos `3.5.0` and a version mismatch may lead to broken packages. But be sure to pin in all packages. Shapely is not pinned. See https://github.com/SciTools/conda-recipes-scitools/blob/master/shapely/meta.yaml#L19